invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -42,14 +44,21 @@ def MChits_and_detsim_electron_simulation_params(nexusMC_filename):
     transverse_diffusion   = 1.00 * units.mm / units.cm**0.5
     longitudinal_diffusion = 0.30 * units.mm / units.cm**0.5
 
-    return xs, ys, zs, energies,\
-           wi, fano_factor, lifetime, drift_velocity,\
-           transverse_diffusion, longitudinal_diffusion
+
+    typename = "MChits_and_detsim_params"
+    field_name = ["xs", "ys", "zs", "energies", "wi", "fano_factor", "lifetime",
+                  "drift_velocity", "transverse_diffusion", "longitudinal_diffusion"]
```

Comment:
Lists of strings are very annoying to type, and quite noisy to read, so avoid doing it ... especially when Python is working hard to make your life easy for you:

```python
In [1]: from collections import namedtuple                                                                                                                                                                                                                                        

In [2]: names = "this that other"                                                                                                                                                                                                                                                 

In [3]: XXX = namedtuple("XXX", names)                                                                                                                                                                                                                                            

In [4]: XXX(1,2,3)                                                                                                                                                                                                                                                                
Out[4]: XXX(this=1, that=2, other=3)
```

Reply:
And if the line gets too long for your liking, triple-quoted strings will work just fine:

```python
In [1]: from collections import namedtuple                                                                                                                                                                                                                                        

In [2]: lots_of_names = """one two three 
   ...:    four five six"""                                                                                                                                                                                                                                                       

In [3]: namedtuple("X", lots_of_names)(1,2,3,4,5,6)                                                                                                                                                                                                                               
Out[3]: X(one=1, two=2, three=3, four=4, five=5, six=6)
```